### PR TITLE
Adding Flare (by GoDaddy) app to the Showcase page

### PR DESCRIPTION
**Pull request by dudeinthemirror**
Adding Flare (by GoDaddy) app to the Showcase page.

Flare is a social network that connects entrepreneurs to fellow entrepreneurs, potential consumers, investors and experts and creates community around new ideas.

Flare was featured both in the AppStore and GooglePlay in 2016

### CODE REVIEW

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -360,6 +360,14 @@ var featured = [
...
     infoTitle: 'Why we chose React Native'
+  },
+  {
+    name: 'Flare',
+    icon: 'http://x.co/FlareIcon',
+    linkAppStore: 'http://x.co/Flare',
+    linkPlayStore: 'http://x.co/FlareAndr',
+    infoLink: 'http://x.co/FlareNews',
+    infoTitle: '"Flare is a social network that connects entrepreneurs to fellow entrepreneurs, potential consumers, investors and experts and creates community around new ideas. The app was featured both in the AppStore and GooglePlay in 2016"',
   }
```
Comment (hramos) [inline website/src/react-native/showcase.js]: Please add a trailing comma here.

File: website/src/react-native/showcase.js [hunk 2]
```diff
@@ -360,6 +360,14 @@ var featured = [
...
     infoLink: 'https://medium.com/causr/why-we-chose-react-native-abd7d58a18b5',
     infoTitle: 'Why we chose React Native'
+  },
+  {
+    name: 'Flare',
+    icon: 'http://x.co/FlareIcon',
+    linkAppStore: 'http://x.co/Flare',
+    linkPlayStore: 'http://x.co/FlareAndr',
+    infoLink: 'http://x.co/FlareNews',
+    infoTitle: '"Flare is a social network that connects entrepreneurs to fellow entrepreneurs, potential consumers, investors and experts and creates community around new ideas. The app was featured both in the AppStore and GooglePlay in 2016"',
```
Comment (hramos) [inline website/src/react-native/showcase.js]: Can you use a shorter title here? Longer titles can look odd in the showcase.

**Comment by dudeinthemirror** [comment]
@hramos thanks for your review. Addressed your feedback in a second commit.

File: website/src/react-native/showcase.js [hunk 3]
```diff
@@ -360,7 +360,15 @@ var featured = [
...
     infoTitle: 'Why we chose React Native'
-  }
+  },
+  {
+    name: 'Flare',
+    icon: 'http://x.co/FlareIcon',
+    linkAppStore: 'http://x.co/Flare',
+    linkPlayStore: 'http://x.co/FlareAndr',
+    infoLink: 'http://x.co/FlareNews',
+    infoTitle: '"Social network that connects entrepreneurs to fellow entrepreneurs, consumers, investors and experts"',
```
Comment (hramos) [inline website/src/react-native/showcase.js]: One last thing, can you remove the double quotes around this title.

**Comment by dudeinthemirror** [comment]
@hramos: Done. Please verify.

**Comment by dudeinthemirror** [comment]
@hramos : could you please help with the import error

**Comment by hramos** [comment]
@dudeinthemirror looking into it

**Pull request closed without merging**